cabal.project

```diff
@@ -1,6 +1,6 @@
 -- Generated by stackage-to-hackage
 
-with-compiler: ghc-8.10.4
+with-compiler: ghc-8.10.7
```

Comment:
This commit, 8881558351ca85313a0746c4be5b40c727292ac0, also lacks `Problem` and `Solution` in its message as well as an issue tag (I think `[Chore]` should be used here).

Reply:
Have included proper message in all commits.